Cargo.toml

```diff
@@ -1,17 +1,30 @@
 [package]
 name = "tree-sitter-lint"
-version = "0.1.0"
+version = "0.0.1-dev.0"
 edition = "2021"
+license = "Unlicense OR MIT"
+authors = [
+  "Julian Rosse <julian@helixbass.net>",
+]
+description = """
+tree-sitter-lint is (intended to be) a multi-language
+linter supporting custom lint rules based on tree-sitter.
+"""
+homepage = "https://github.com/helixbass/tree-sitter-lint"
+repository = "https://github.com/helixbass/tree-sitter-lint"
+keywords = ["tree-sitter", "lint", "linter"]
+categories = ["command-line-utilities", "development-tools"]
+rust-version = "1.70"
 
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
 clap = "4.3.17"
 derive_builder = "0.12.0"
-proc_macros = { path = "proc_macros" }
+proc_macros = { package = "tree_sitter_lint_proc_macros", path = "proc_macros", version = "0.0.1-dev.0" }
 regex = "1.9.1"
 tree-sitter = "0.20.10"
-tree-sitter-grep = { git = "https://github.com/helixbass/tree-sitter-grep", rev = "3d4682c" }
+tree-sitter-grep = { package = "tree_sitter_lint_tree-sitter-grep", git = "https://github.com/helixbass/tree-sitter-grep", rev = "422058d", version = "0.1.0" }
```

Comment:
I guess if/when the `tree-sitter-grep` crate-level API "lands" (+ gets published) this could switch over to using the "normal" package but for now published a version of it that's specific to `tree-sitter-lint`